src/ipc.rs

```diff
@@ -73,7 +73,7 @@ thread_local! {
 /// [IpcReceiver]: struct.IpcReceiver.html
 pub fn channel<T>() -> Result<(IpcSender<T>, IpcReceiver<T>),Error>
                   where T: for<'de> Deserialize<'de> + Serialize {
-    let (os_sender, os_receiver) = try!(platform::channel());
+    let (os_sender, os_receiver) = ?(platform::channel());
```

Comment:
Should be just `platform::channel()?`. no parenthesis around it and the `?` after.